src/core/bau_inputs.jl

```diff
@@ -32,7 +32,7 @@ function BAUInputs(p::REoptInputs)
     cooling_cf = Dict{String, Array{Float64,1}}() 
     avoided_capex_by_ashp_present_value = Dict(t => 0.0 for t in techs.all)
     production_factor = DenseAxisArray{Float64}(undef, techs.all, p.time_steps)
-    tech_renewable_energy_fraction = Dict(t => 0.0 for t in techs.all)
+    tech_renewable_energy_fraction = Dict(t => 0.0 for t in union(techs.fuel_burning, techs.elec))
     # !!! note: tech_emissions_factors are in lb / kWh of fuel burned (gets multiplied by kWh of fuel burned, not kWh electricity consumption, ergo the use of the HHV instead of fuel slope)
     tech_emissions_factors_CO2 = Dict(t => 0.0 for t in techs.all)
```

Comment:
@zolanaj  in the `if "Generator" in techs.all` statement, do you think we should add this?: 

`tech_renewable_energy_fraction["Generator] = p.s.generator.fuel_renewable_energy_fraction`

Reply:
> @zolanaj in the `if "Generator" in techs.all` statement, do you think we should add this?:
> 
> `tech_renewable_energy_fraction["Generator] = p.s.generator.fuel_renewable_energy_fraction`

Agreed, good catch @adfarth! I addressed this in https://github.com/NREL/REopt.jl/commit/cc3abd0113f92e0f14b84a80ef456e58d9038bba